platform/core/src/services/UIModalService/index.js

```diff
@@ -38,19 +35,16 @@ function createUIModalService() {
  * Show a new UI modal;
  *
  * @param {Modal} component React component
- * @param {ModalProps} props { header, footer, backdrop, keyboard, show, closeButton, title, customClassName }
+ * @param {ModalProps} props { shouldCloseOnEsc, isOpen, closeButton, title, customClassName }
  */
 function show(
   component,
   props = {
-    header: null,
-    footer: null,
-    backdrop: false,
-    keyboard: false,
-    show: true,
```

Comment:
👀 Goodbye? 👋 
Not disagreeing, but curious to hear reasoning.

Reply:
My first guess is simply that we're not using these (yet) (KISS) (YAGNI)

Reply:
Yes, we're not using them yet but I think header a footer being passed as props is just weird, header and footer are content's responsibility now.